doc/install/python.rst

```diff
@@ -117,6 +108,12 @@ The following flags can be used to configure the installation:
    and ``CMake`` under the hood, so all flags and options valid in ``CMake`` can
    be used in this fashion.
 
+   Allthough the `scikit-build documentation <https://scikit-build.readthedocs.io/en/latest/usage.html#environment-variable-configuration>`_
+   mentions that you can also pass the build options with ``--install-option=""``, this will cause ``pip`` to build
+   all dependencies, including all build-dependencies, instead of downloading them from PyPI. ``CMAKE_ARGS=""``
+   saves you the build time, and also downloading and setting up the dependencies they in turn require to be present.
+   Setting ``CMAKE_ARGS=""`` is in addition compatible with build front-ends like `build <https://pypa-build.readthedocs.io>`_.
+
```

Comment:
line length? There's usually a keyboard shortcut to linewrap a paragraph

Reply:
It's consistent with some other line lengths in this file, but not all, correct. I'll break the again at `.`'s and `'`'s.